src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -4389,60 +4405,42 @@ TEST_CASE("SpatialDropoutLayerTest", "[ANNLayerTest]")
   SpatialDropout<> module(3, 0.2);
 
   // Input is a batch of 2 images, each of size (2,2) and having 4 channels.
-  input << 0.4963 << 0.0885 << 0.7682 << 0.1320 << 0.3074 << 0.4901 << 0.6341
-      << 0.8964 << 0.4556 << 0.3489 << 0.6323 << 0.4017 << arma::endr;
+  input = { 0.4963 , 0.0885 , 0.7682 , 0.1320 , 0.3074 , 0.4901 , 0.6341 , 0.8964 , 0.4556 , 0.3489 , 0.6323 , 0.4017 };
```

Comment:
```suggestion
  input = { 0.4963 , 0.0885 , 0.7682 , 0.1320 , 0.3074 , 0.4901 , 0.6341 ,
      0.8964 , 0.4556 , 0.3489 , 0.6323 , 0.4017 };
```
There seem to be few lines which are longer than 80 characters. 
You could continue them in a new line. 

For more details on this, refer the style guide 
https://github.com/mlpack/mlpack/wiki/DesignGuidelines#line-length-and-wrapping

Reply:
Good job on the line wrapping. You missed out the line pointed out here.